Replace U+0332 by U+0331

moyogo opened the pull request:
In Yaghoubi romanization ږ should be romanized with ẕ̌ (1E95 LATIN SMALL LETTER Z WITH LINE BELOW, 030C COMBINING CARON) as U+0331 COMBINING MACRON BELOW is used in all other letter with line below cases, instead of with U+0332 COMBINING LOW LINE.

moyogo committed:
Replace U+0332 by U+0331

In Yaghoubi romanization ږ should be romanized with ẕ̌ (1E95 LATIN SMALL LETTER
Z WITH LINE BELOW, 030C COMBINING CARON) as U+0331 COMBINING MACRON BELOW is
used in all other letter with line below cases, instead of with U+0332
COMBINING LOW LINE.

ronaldtse commented:
Thank you @moyogo ! The test failures are not related to the changes, and the change is good to go. 

You are probably correct that U+0332 combining caron should have used instead of U+0331 for all characters in this map. Would you be able to extend the PR to fix those as well? Thanks!

moyogo committed:
Replace U+0332 by U+0331

U+0331 is used in precomposed characters like "ḵ" ("क़", "ḵa") when
decomposed. This changes composed characters like "g̲" ("ग़", "g̲a")
to "g̱" using U+0331 instead of U+0332, , or composed characters like
"h̲" ("ह़", "ẖa") to precomposed characters like "ẖ".

moyogo commented:
@ronaldtse I’ve add a commit with the DIN 33904-2018 changes.
